### ISSUE TITLE
`CouchDbSessionAuthenticator` does not respect `${SERVICE}_AUTH_DISABLE_SSL` variable

### ISSUE THREAD

**Issue by eiri** [IBM/cloudant-python-sdk#326]
**Describe the bug**

When authenticator set to `CouchDbSessionAuthenticator` it throws an exception on self-signed cert even when `CLOUDANT_DISABLE_SSL` variable set to `true`.

There are new env var `${SERVICE}_AUTH_DISABLE_SSL` in core-sdk that allows to skip cert validation for authenticators, but `CouchDbSessionAuthenticator` doesn't support it.

**To Reproduce**

Start local couch with self-signed cert and run a simple db info call with env file looking like this:

```
CLOUDANT_AUTH_TYPE=COUCHDB_SESSION
CLOUDANT_URL=https://127.0.0.1:5984
CLOUDANT_USERNAME=xxx
CLOUDANT_PASSWORD=xxx
CLOUDANT_DISABLE_SSL=true
CLOUDANT_AUTH_DISABLE_SSL=true
```

**Expected behavior**

Client should skip cert validation and return db info. Actually an exception for invalid cert is thrown.

**Must gather (please complete the following information):**
 - SDK Version 0.2.0
 - Python Version 3.10

**Additional context**

The root cause here is that originally core used the same var `${SERVICE}_DISABLE_SSL` to skip cert validation for both authenticator and main service, but since in the majority of the cases those two are different services they split the var at some moment and implementation of `CouchDbSessionAuthenticator` predate this.

Conceptually `CouchDbSessionAuthenticator` is in a special position, since it is always uses the same URL (it is theoretically possible to have couch session as an external service, but that's pretty arcane). So I think fix also should include an automatic behavior that when `${SERVICE}_DISABLE_SSL` set to `true` it is also extends to session authenticator. 

Naturally this fix has a pretty low priority, since this is a controversial feature that allows inadvertently degrade a production service's security and there are no real driver to use SSLin dev process, not until we start to support http/2
